### DISCUSSION TITLE
What is the differences of this image to mongodb-community-server ?

### DISCUSSION THREAD

**Discussion by sibelius** [docker-library/mongo#642] [General]
what are the differences from

https://hub.docker.com/_/mongo

and

https://hub.docker.com/r/mongodb/mongodb-community-server

**Comment by tianon** [comment]
I think https://explore.ggcr.dev/layers/mongodb/mongodb-community-server@sha256:d968886ca3dd15bee32eda24f6c8559cdb1c1d1f0ddd9c6a6cbe44b191ec5e7f/usr/local/bin/docker-entrypoint.py is probably going to be the primary difference (they've ported the entrypoint to Python).

**Reply by tianon** [reply]
The other main difference being that the `mongodb/mongodb-community-server` image is maintained directly by MongoDB employees.
